src/JWT.php

```diff
@@ -24,30 +23,31 @@ class JWT
      * @param string $origin Optionally limit header origin
```

Comment:
`$expiry` should be added to the PHPDoc comment.

Reply:
Also, might it not be better to make the expiry parameter the timeout, instead of a timestamp? Then it can get a default value of `3600`. Might simplify it slightly.